README.md

```diff
@@ -1,4 +1,5 @@
 # edx-api-client
+[![codecov.io](https://codecov.io/github/codecov/codecov-ruby/coverage.svg?branch=master)](https://codecov.io/github/codecov/codecov-ruby?branch=master)
```

Comment:
I think you need to customize this to point to our repo. 

[![codecov.io](https://codecov.io/github/mitodl/edx-api-client/coverage.svg?branch=master)](https://codecov.io/github/mitodl/edx-api-client?branch=master)

```
[![codecov.io](https://codecov.io/github/mitodl/edx-api-client/coverage.svg?branch=master)](https://codecov.io/github/mitodl/edx-api-client?branch=master)
```